pci/src/vfio.rs

```diff
@@ -357,14 +357,14 @@ pub(crate) struct VfioCommon {
     pub(crate) interrupt: Interrupt,
     pub(crate) msi_interrupt_manager: Arc<dyn InterruptManager<GroupConfig = MsiIrqGroupConfig>>,
     pub(crate) legacy_interrupt_group: Option<Arc<dyn InterruptSourceGroup>>,
+    pub(crate) vfio_wrapper: Arc<dyn Vfio>,
```

Comment:
I'm sure there was a reason why I didn't do this originally. But since it obviously works then :+1:

Reply:
Yeah I don't know, but it's cool that it works as this simplifies things a bit, and it will be useful for restoring the device.